scripts/create_bridges.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service to create bridges
+Before=libvirtd.service
+After=network.target
```

Comment:
Okay. Do you think we should have a ``Wants=network-online.target`` dependency too? 
And while we are at it, should we modify [haas_network.service](https://github.com/CCI-MOC/hil/blob/master/scripts/haas_network.service) as well?